### PR TITLE
[Add]: add fossil R package

### PR DESCRIPTION
**Pull request by qdu-bioinfo**
Add fossil R package.

### CODE REVIEW

**Comment by qdu-bioinfo** [comment]
@conda-forge-bot please add label

**Comment by qdu-bioinfo** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by qdu-bioinfo** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by qdu-bioinfo** [comment]
@conda-forge-admin  please ping team

File: recipes/r-fossil/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: http://matthewvavrek.com/programs-and-code/fossil/
+  license: GPL-2
```
Comment (dbast) [inline recipes/r-fossil/meta.yaml:57]:
```suggestion
  license: GPL-2.0-or-later
```

File: recipes/r-fossil/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fossil_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fossil/fossil_{{ version }}.tar.gz
+  sha256: 37c082fa15ebae89db99d6071b2bb2cad6a97a0405e9b4ef77f62a8f6ad274c1
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
```
Comment (dbast) [inline recipes/r-fossil/meta.yaml:18]: Please remove the comments except of the cran meta data or rerun with newest conda-build skeleton cran and the `--no-comments` option. thx.

File: recipes/r-fossil/meta.yaml (hunk 1 above)
Comment (qdu-bioinfo) [reply recipes/r-fossil/meta.yaml:46]: I tried this before, but this makes the recipe fail in macOS test, I don't know why....

**Comment by dbast** [comment]
@qdu-bioinfo Thanks and welcome to conda-forge.

**Pull request merged**